capstone-app/src/app/modules/history/history.module.ts

```diff
@@ -1,10 +1,16 @@
 import { NgModule } from '@angular/core';
 import { CommonModule } from '@angular/common';
+import { WikiMediaComponent } from './wiki-media/wiki-media.component';
 
 @NgModule({
-  declarations: [],
+  declarations: [WikiMediaComponent],
   imports: [
     CommonModule
   ]
```

Comment:
Have you verified that you can use WikiMediaComponent? You may need to create an array of exports here and export the component.

Reply:
Why would I be unable to use that component?

Reply:
Because you created it in a different module outside of the root module (app.module.ts). If you don't export it then I believe you will be unable to use it in app.component.html